templates/case-studies/templates/_macros-hero.jinja

```diff
@@ -0,0 +1,33 @@
+# Params
+# title: Hero title text (required)
+# subtitle: Hero subtitle text
+# logo: Path to logo image
+# logo_alt: Alt text for logo image
+# logo_width: Width of logo image (128 by default)
+# logo_height: Width of logo image (128 by default)
+
+{%- macro hero(
+  title,
+  subtitle,
+  logo,
+  logo_alt="",
+  logo_width="128",
+  logo_height="128"
+  ) -%}
+  <section class="p-section--hero">
+    <div class="row--25-75">
+      <div class="col">
+        <div class="p-image-wrapper">
+          <img src={{ logo }} alt="{{ logo_alt }}" width="{{ logo_width }}" height="{{ logo_height }}" />
```

Comment:
I think so far we've been treating images as slots in our patterns.

So instead of passing all the different image attributes directly, you pass HTML of the whole image element.
This for example would allow to use the web team image template to serve image via cloudinary instead of it being directly defined as here.

Reply:
I will be using the current Vanilla macro for hero section for this, will keep slots usage in mind for future works.